subcell_pipeline/simulation/batch_simulations.py

```diff
@@ -0,0 +1,230 @@
+import re
```

Comment:
Should this file be in the `cytosim` folder since the functions are specific to cytosim?

Reply:
Okay with leaving it here for now and revisiting once we have the ReaDDy workflows refactored!